assets/src/block-editor/plugins/amp-toggle.js

```diff
@@ -2,7 +2,7 @@
 /**
  * External dependencies
  */
-import { errorMessages, isStandardMode } from 'amp-block-editor-data';
+import { errorMessages } from 'amp-block-editor-data';
```

Comment:
@todo: there's a console error related to `amp-block-editor-data`, though probably not caused by this line:

```js
Uncaught ReferenceError: ampBlockEditor is not defined
    at Object.amp-block-editor-data (amp-block-validation.js:5180)
    at __webpack_require__ (amp-block-validation.js:20)
    at Module../assets/src/block-editor/components/amp-preview.js (amp-block-validation.js:114)
    at __webpack_require__ (amp-block-validation.js:20)
    at Module../assets/src/block-editor/components/index.js (amp-block-validation.js:471)
    at __webpack_require__ (amp-block-validation.js:20)
    at Module../assets/src/block-editor/helpers/index.js (amp-block-validation.js:907)
    at __webpack_require__ (amp-block-validation.js:20)
    at Module../assets/src/block-validation/index.js (amp-block-validation.js:2512)
    at __webpack_require__ (amp-block-validation.js:20)
```